src/indexHtml.mustache

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
```

Comment:
This file was adapted from https://github.com/single-spa/create-single-spa/blob/main/packages/generator-single-spa/src/root-config/templates/src/index.ejs